sdk/test/NetStandard/IntegrationTests/IntegrationTests/EC2/BasicDescribes.cs

```diff
@@ -48,7 +52,7 @@ public async System.Threading.Tasks.Task TestDescribeAmazonImagesCancellationTes
             });
 
             //Assert.Equal(token, exception.CancellationToken);
-            Assert.Equal(true, exception.CancellationToken.IsCancellationRequested);
+            Assert.True(exception.CancellationToken.IsCancellationRequested);
```

Comment:
Nice, there are so many new (at least since .NET 8) xUnit warnings in the .NET Standard tests...